pkg/bugreport/controlplane.go

```diff
@@ -0,0 +1,30 @@
+package bugreport
+
+import (
+	"context"
+	"fmt"
+	"path"
+	"strings"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (c *Config) collectControlPlaneLogs() error {
+	pods, err := c.KubeClient.CoreV1().Pods(c.ControlPlaneNamepace).
+		List(context.Background(), v1.ListOptions{
+			LabelSelector: "app.kubernetes.io/name=openservicemesh.io",
```

Comment:
Nit: we have constants defined for this key and value:
https://github.com/openservicemesh/osm/blob/9b505557cfa0135a4e352a89f84e606f8aa9d9da/pkg/constants/constants.go#L202-L203

Reply:
Good catch! I fixed it, thanks!